RPM_CHANGES

```diff
@@ -1,6 +1,11 @@
 # Machinery RPM Changelog
 
 
+## Version 1.3.0 - Mon Jan 26 13:07:06 CET 2015 - chuller@suse.de
+
+* update to version 1.3.0
```

Comment:
you just need to add the tracking but, everything else is done by our release script so the above part needs to be removed